### PR TITLE
Add server

### PR DESCRIPTION
**Pull request by Fln1k**

### CODE REVIEW

**Review by shukal94 — CHANGES_REQUESTED** [review]
In this repository we don't need at all. Reason: I'm not going to change your own virtualenv and pycharm and pycache files, we need only source code. 
Suggest to customize `.gitignore` file

File: main.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            draw.rectangle(
+                (
+                    (pointer_x)*zoom,
+                    (pointer_y)*zoom,
+                    (pointer_x+1)*zoom,
+                    (pointer_y+1)*zoom
+                ),
+                fill=dict[point_columns]
+            )
+    img.save('Схема.png')
```
Comment (shukal94) [inline main.py]: In future, try to avoid Cyrillic codecs, servers where you going to deploy might be not support russian locale

File: main.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿from flask import Flask, request, json
+from PIL import Image, ImageDraw
+
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=['GET', 'POST'])
+def make_matrix():
```
Comment (shukal94) [inline main.py]: which matrix?

File: main.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    a = request.json
```
Comment (shukal94) [inline main.py]: you can try `a = request.get_json()`
and what is `a`

File: main.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    dict = {'Not_Available': 'red', 'Available': 'green', 'Cashbox': 'yellow', 'Rack': 'blue'}
```
Comment (shukal94) [inline main.py]: 1) `dict` - reserved python identifier, choose other name,
2) could you please move states dictionary into another place, more global?

File: main.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    img = Image.new('RGB', (1400, 1000), color='white')
```
Comment (shukal94) [inline main.py]: remove hardcode

File: main.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    draw = ImageDraw.Draw(img)
+    pointer_y = 0
```
Comment (shukal94) [inline main.py]: pointer translates as `указатель`

File: main.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=['GET', 'POST'])
+def make_matrix():
+    a = request.json
+    dict = {'Not_Available': 'red', 'Available': 'green', 'Cashbox': 'yellow', 'Rack': 'blue'}
+    img = Image.new('RGB', (1400, 1000), color='white')
+    draw = ImageDraw.Draw(img)
```
Comment (shukal94) [inline main.py:24]: please move image processing logic into another place

**Review by shukal94 — CHANGES_REQUESTED** [review]
you have a GOD endpoint there, not a good practice

File: main.py (hunk 6 above)
Comment (Fln1k) [reply main.py]: указатель по x, указатель по y. what's wrong?

File: main.py [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+
+app = Flask(__name__)
+zoom = 3
+
+
+@app.route("/", methods=['GET', 'POST'])
+def draw_scheme():
+    state_matrix = json.request
+    state_dictionary = {'Not_Available': 'red',
```
Comment (shukal94) [inline main.py]: you can place state dict outside of any functions, it's constant so you'll don't need to pass whis as parameter

**Review by shukal94 — APPROVED** [review]

**Pull request merged**